setup.cfg

```diff
@@ -11,6 +11,8 @@ include_package_data = true
 install_requires =
     whitenoise == 5.2.0
     django-pipeline == 2.0.6
+    libsass == 0.21.0
+    jsmin==2.2.2
```

Comment:
Are we expecting JSmin in WDIV?

Reply:
I was getting the following error running `collectstatic` in WCIVF since we introduced `dc_utils`:
```
Type 'yes' to continue, or 'no' to cancel: yes
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 188, in handle
    collected = self.collect()
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 128, in collect
    for original_path, processed_path, processed in processor:
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/pipeline/storage.py", line 37, in post_process
    packager.pack_javascripts(package)
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/pipeline/packager.py", line 122, in pack_javascripts
    return self.pack(package, self.compressor.compress_js, js_compressed, templates=package.templates, **kwargs)
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/pipeline/packager.py", line 116, in pack
    content = compress(paths, **kwargs)
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/pipeline/compressors/__init__.py", line 66, in compress_js
    js = getattr(compressor(verbose=self.verbose), 'compress_js')(js)
  File "/Users/michaelcollins/code/dc/WhoCanIVoteFor/env/lib/python3.6/site-packages/pipeline/compressors/jsmin.py", line 10, in compress_js
    from jsmin import jsmin
ModuleNotFoundError: No module named 'jsmin'
```

So although I don't know if WDIV uses it, I was presuming that wherever we use dc_utils we would need `jsmin`

Reply:
Sorry, I meant WCIVF!

Also, I see _this project_ specifies it: https://github.com/DemocracyClub/dc_django_utils/blob/main/dc_utils/settings/pipeline.py#L40

So, good spot!